pkgs/development/compilers/rust/rustc.nix

```diff
@@ -109,6 +109,9 @@ in stdenv.mkDerivation (finalAttrs: {
     "--tools=rustc,rustdoc,rust-analyzer-proc-macro-srv"
     "--enable-rpath"
     "--enable-vendor"
+    # For Nixpkgs it makes more sense to use stdenv's linker than
+    # letting rustc build its own.
+    "--disable-lld"
```

Comment:
Maybe this is worth splitting into a different commit/what made you change this with this bump?

Reply:
It started complaining if I didn't set it.

Reply:
So it's not a change, it's preserving the previous behaviour.